src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style the elements using `.classes`. In this case, consider adding a class to the `ul` element in the HTML and style that class instead.